src/pages/CertificateViewer.js

```diff
@@ -171,6 +180,7 @@ export default function CertificateViewer() {
         <Grid.Panel area="main">
           <ContainerDiv>
             <Paper>
+              {/* TODO: Add some {loading} spinner */}
```

Comment:
This can be removed now?

Reply:
Yes, i think i can just remove it .. it would have been nice to have some design for this but i guess we need to keep what we have ..